src/Conf.ml

```diff
@@ -454,6 +455,20 @@ module Formatting = struct
     C.choice ~names ~all ~env ~doc ~allow_inline:true ~update:(fun conf x ->
         {conf with let_and= x} )
 
+  let let_binding =
```

Comment:
Oh, and at this point, this name is too specific, maybe `structure_item_grouping` instead?